### PR TITLE
[W5.4][T15-A4]Archana Pradeep

### PR DESCRIPTION
**Pull request by archthegit**
ready for review

### CODE REVIEW

**Comment by moomeowroar** [comment]
good job

File: doc/UserGuide.md [hunk 1]
```diff
@@ -57,14 +57,14 @@ Format: `list`
...
+> The search is case in-sensitive, the order of the keywords does not matter, only the name is searched, 
 and persons matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples: 
 * `find John`<br>
-  Returns `John Doe` but not `john`
+  Returns `John Doe` and `john`
 * `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+  Returns Any person having names `Betsy`, `Tim`, and `John`
```
Comment (okkhoy) [inline doc/UserGuide.md]: err.. did you make an `or` search to an `and` search?

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -1,10 +1,7 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
+import java.lang.String;
```
[the remark this answers is not in the document]
Comment (okkhoy) [reply src/seedu/addressbook/commands/FindCommand.java]: @CT15 made my job easy! @archthegit please take note :smile:

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -48,8 +45,20 @@ public CommandResult execute() {
...
+            keywordsIntermediate[i]=keywordsIntermediate[i].toLowerCase();
+        }
+        keywords.clear();
+        keywords.addAll(Arrays.asList(keywordsIntermediate));
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            final List<String> wordsIntermediate= person.getName().getWordsInName();
+            ListIterator<String> wordIterator = wordsIntermediate.listIterator();
+            while (wordIterator.hasNext())
+            {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java]: violation of coding standards here. please check how/where should braces be written

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 4]
```diff
@@ -26,7 +26,7 @@ public void execute() throws IllegalValueException {
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
         //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        assertFindCommandBehavior(new String[]{"aMy"},Arrays.asList(td.amy));
```
Comment (okkhoy) [inline test/java/seedu/addressbook/commands/FindCommandTest.java]: need space after the `,`

**Comment by okkhoy** [comment]
@archthegit 
Some comments added. Please ack & close the PR after reading comments.
@CT15 Good job on the review!

File: doc/UserGuide.md (hunk 1 above)
Comment (archthegit) [reply doc/UserGuide.md]: yeah i can search for multiple people and they all show up

**Pull request closed without merging**